src/main/java/stream/flarebot/flarebot/mod/Punishment.java

```diff
@@ -43,7 +43,9 @@ public MessageEmbed getActionEmbed(User user, User responsible, String reason, b
         eb.setTitle(getName());
         eb.setColor(Color.WHITE);
         eb.addField("User", user.getName() + "#" + user.getDiscriminator() + " (" + user.getId() + ")", true);
-        if (responsible != null)
+        if(action == ModlogAction.REPORT_CREATE)
```

Comment:
If we did this for everything the code would be a mass of ifs. Either find a better way to handle this or use the EmbedBuilder option.

Reply:
Fair enough